src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,63 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final String DELIMITER = ",";
+    private static final int OPERATION_TYPE_INDEX = 0;
+    private static final int AMOUNT_INDEX = 1;
 
+    public void getStatistic(String fromFileName, String toFileName) {
+        StringBuilder resultBuilder = new StringBuilder();
+        File formFile = new File(fromFileName);
+        File toFile = new File(toFileName);
+        try (BufferedReader reader = new BufferedReader(new FileReader(formFile))) {
```

Comment:
```suggestion
        try (BufferedReader reader = new BufferedReader(new FileReader(formFileName))) {
```
you don't need files